src/pages/Integrations/SplunkSetup/useSplunkSetup.tsx

```diff
@@ -80,8 +80,18 @@ export const useSplunkSetup = () => {
   const attachEvents = useAttachEventsToSplunk();
 
   return async ({ hecToken, splunkServerHostName }, onProgress) => {
-    const integrationName = SPLUNK_INTEGRATION_NAME;
-    const behaviorGroupName = SPLUNK_BEHAVIOR_GROUP_NAME;
+    const now = new Date();
+    const timestamp = `${
+      now.getDate().toString().padStart(2, "0")
+      }${(now.getMonth() + 1).toString().padStart(2, "0")
+      }${now.getFullYear()
+      }${now.getHours().toString().padStart(2, "0")
+      }${now.getMinutes().toString().padStart(2, "0")
+      }${now.getSeconds().toString().padStart(2, "0")
+    }`;
```

Comment:
No need to do complex calculation of timestamp. You can replase this with `getTime` function
```suggestion
    const timestamp = now.getTime();
```

Reply:
Ow.. nice!! thanks!